Bio/PDB/Atom.py

```diff
@@ -482,6 +483,27 @@ def __repr__(self):
         """Return disordered atom identifier."""
         return "<Disordered Atom %s>" % self.get_id()
 
+    # This is a separate method from Entity.center_of_mass since DisorderedAtoms
+    # will be unpacked by Residue.get_unpacked_list(). Here we allow for a very
+    # specific use case that is much simpler than the general implementation.
+    def center_of_mass(self, geometrical=False):
```

Comment:
Do you have strong views on ``geometical`` versus ``geometric`` here?

Reply:
Not at all. Changed it!

Reply:
Thanks - the shorter works seems clearer to me.